test/integration/rest.test.js

```diff
@@ -441,7 +441,7 @@ describe('REST authentication', function () {
     });
 
     describe('when authentication succeeds', () => {
-      it('emits login event', done => {
+      it.only('emits login event', done => {
```

Comment:
This is the failing test.  Well, one of two that are failing for what looks like the same cause.